ERCS/erc-7566.md

```diff
@@ -0,0 +1,193 @@
+---
+eip: 7566
+title: Multiplayer Onchain Game
```

Comment:
Your title sounds more like you are standardizing one particular game. You should try to make it more general.

Reply:
Hey @SamWilsn , I want to discuss more, this name is the same as Multi Token Standard (erc1155). It does not specifically refer to the type of game Multiplayer Online Game. Instead, it refers to Multi Player Standard. Prevent confusion between Online and Onchain. Do you think Multiplayer Game Standard is ok?

Reply:
Yes, I see; "Multiplayer Game Standard" sounds better than just "Multiplayer Game". How about something like "Multiplayer Game Rooms" or "Multiplayer Game Communication"?

Reply:
Hey @SamWilsn , thank you so much for checking again, "Multiplayer Game Communication" is so cool, I used it, btw, I already created a game with this proposal, https://chainchess.xyz/